inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/HeartbeatManager.java

```diff
@@ -138,8 +139,9 @@ private Runnable heartBeatReportThread() {
                     int heartbeatInterval = conf.getInt(AGENT_HEARTBEAT_INTERVAL,
                             DEFAULT_AGENT_HEARTBEAT_INTERVAL);
                     TimeUnit.SECONDS.sleep(heartbeatInterval);
-                } catch (Exception ex) {
+                } catch (Throwable ex) {
                     LOGGER.error("error caught", ex);
```

Comment:
There is also log processing in the threadThrowableHandler method. Can the log output in the catch later be placed in it?

Reply:
> There is also log processing in the threadThrowableHandler method. Can the log output in the catch later be placed in it?

There may be  some other error messages, such as 'LOGGER.error("error while init sender for group id", ex)'.  Moving it  into threadThrowableHandler method does not reflect some key information.